ProjectLighthouse.Servers.GameServer/Controllers/Slots/SlotsController.cs

```diff
@@ -270,15 +272,17 @@ public async Task<IActionResult> TeamPickedSlots([FromQuery] int pageStart, [Fro
     }
 
     [HttpGet("slots/lbp2luckydip")]
-    public async Task<IActionResult> LuckyDipSlots([FromQuery] int pageStart, [FromQuery] int pageSize, [FromQuery] int seed)
+    public async Task<IActionResult> LuckyDipSlots([FromQuery] int pageStart, [FromQuery] int pageSize, [FromQuery] int seed) //here
```

Comment:
Remove un-necessary comment
```suggestion
    public async Task<IActionResult> LuckyDipSlots([FromQuery] int pageStart, [FromQuery] int pageSize, [FromQuery] int seed)
```